integrations/sonarqube/main.py

```diff
@@ -28,14 +21,13 @@ def init_sonar_client() -> SonarQubeClient:
 @ocean.on_resync(ObjectKind.PROJECTS)
 async def on_project_resync(kind: str) -> list[dict[str, Any]]:
     logger.info(f"Listing Sonarqube resource: {kind}")
+
     sonar_client = init_sonar_client()
     return await sonar_client.get_all_projects()
 
 
 @ocean.on_resync(ObjectKind.ISSUES)
 async def on_issues_resync(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
-    logger.info(f"Listing Sonarqube resource: {kind}")
```

Comment:
Why did you remove all the logs?

Reply:
The ocean framework already logs these resync event for each kind. So adding another log makes it redundant